packages/doc/content/components/components/stepper/stepper-web.mdx

```diff
@@ -4,36 +4,60 @@ Gympass `<Stepper />` component is used to guide the user through determined ste
 
 ### Usage
 
+You can use it with or without children.
+
 #### Simple
 
-```javascript reactLive=true center=true
-<Stepper activeStep={0}>
-  <Stepper.Step label="Confirm Booking" />
-  <Stepper.Step label="Class Booked" />
-  <Stepper.Step label="Check-in" />
-</Stepper>
-```
+```javascript reactLive=true center=true state=true
+render(() => {
+  const [activeStep, setActiveStep] = useState(0);
```

Comment:
Could you add a break line between declarations?
💅

Reply:
Done!